### ISSUE TITLE
Poedit doesn’t respect (dark) theme colors under GTK+

### ISSUE THREAD

**Issue by nikoss** [vslavik/poedit#273]
Hallo,

I just tried Poedit 1.8.7 with a dark theme and I ran into 2 problems.

I. The sidebar font has similar colour as the background.

II. After typing something like " Ctr+2", to choose a suggested translation,
now, the text, in the box with the translated string
(that is, the text that was just added),
also changes colour to something similar to the background.

I am attaching two picture demonstrating the issue.

My system is...
Mageia 5
32bit
Poedit 1.8.7
IceWM
Gtk3 theme: BlackMate

The BlackMate theme is quite "standard", so I guess this is probably a Poedit issue.

**Comment by nikoss** [comment]
![dark_theme_001](https://cloud.githubusercontent.com/assets/4921343/15716302/6eba1ca8-282a-11e6-8c0f-90f26340176d.jpg)
![dark_theme_002](https://cloud.githubusercontent.com/assets/4921343/15716303/6ecb3eca-282a-11e6-9e5f-fb5043d94296.jpg)

**Comment by vslavik** [comment]
> The BlackMate theme is quite "standard”,

Well no, not really — never header of it, and *all* of your system is pretty crazily non-standard… That doesn’t mean it isn’t a bug, of course, but it decreases its priority.

An itemized list of specific issues would be nice. You posted some screenshots without any comments now, but aside from sidebar, it’s unclear which of the ugliness there is normal for the theme and which things *exactly* are deviations. What’s up with two different text colors (with no explanation about the different in state) for example?

(And of course, as always, patches would be even nicer!)

**Comment by vslavik** [comment]
Ah, never mind, got lost in the original comment’s formatting...

**Comment by nikoss** [comment]
"never header of it " ?
https://github.com/mate-desktop/mate-themes/tree/master/desktop-themes/BlackMATE

quote:
"and all of your system is pretty crazily non-standard"

With " Poedit 1.8.7"... I think you are right ;) .

**Comment by vslavik** [comment]
> "never header of it “ ?

Yes, this may shock you, but I really never heard about some of the more obscure themes used by more obscure DEs before. No, that doesn’t mean I do not believe in its existence and that you have to provide a link. 

Please limit comments to things that contribute to fixing.

**Comment by Getron** [comment]
i have tested Poedit on many distros, KDE 5.x, standard dark themes, but on all it sems to have this issue with background there and the editable text problem.

**Comment by vslavik** [comment]
> on many distros

Doubtful about “many”. But sure, I don’t doubt reproducibility. Your PR would be welcome, but please don’t spam the issue tracker with pointless noise: first a duplicate issue, then this comment that adds nothing to the discussion.

If you contribute, great, it would be most welcome. If you don’t, that’s fine too, but at least please don’t take developers time from actual work. Thank you.

**Comment by Getron** [comment]
I don't spam anything, i just confirm this problem to. I have tested it on "many" distros: fedora, manjaro, linux mint, antergos, kaos linux, and so on..... changed the theme to dark, and the problem is there, on all of this distros. And this problem is presistent long time ago, i think about a year or maybe more, i have tested first time the Poedit on dark theme, and it was there. Today, this problem still there.

**Comment by noirbizarre** [comment]
I don't think this is specific to `BlackMate` but to any dark theme. Here the 3 I'm using:
- Arc Dark ![capture d ecran de 2017-09-16 17-50-48](https://user-images.githubusercontent.com/15725/30513724-951122f0-9b08-11e7-8212-ab90e776e1cc.png)
- Vertex Dark ![capture d ecran de 2017-09-16 17-52-49](https://user-images.githubusercontent.com/15725/30513726-95167f2a-9b08-11e7-9eca-471c99886fdb.png)
- Adwaita Dark ![capture d ecran de 2017-09-16 17-53-19](https://user-images.githubusercontent.com/15725/30513725-9515b2b6-9b08-11e7-81ea-6cf3fcd9dbe3.png)

I also tested the Ubuntu dark one, same issue.

I think the issues is simply that white background color is hard coded on some widgets where it shouldn't (Every text entries, right sidebar...). This make the app barely useable with a dark theme because the cursor is white on white when editing entries. Suggestions (or any content that is in the right sidebar) is not readable to too.
**Consequence:** its really exhausting when you have thousands of translations to handle, you are searching for the cursor to fix an entry, or trying to use a suggestion.

(I would have contributed but I'm no C++ nor wxGTK developper)

PS: @vslavik no offense but I agree with @nikoss , even if `BlackMate` in particular is not a standard theme, having a dark theme is now standard on Linux (even Ubuntu come with one by default), the 3 ones I provided as example are some of the most used theme under gnome) and so it's not really a edge case.

PS2: I'm glad to help in any way if I can

**Comment by vslavik** [comment]
> I don't think this is specific to BlackMate

That's kind of obvious from all the references right above your comment; consequently, that super-long comment is unnecessary and adds precious little to the issue; please limit such noise.

> (I would have contributed but I'm no C++ nor wxGTK developper)

Nobody is *born* one, you know...

**Comment by vslavik** [comment]
Version 2.2 officially supports dark themes: https://poedit.net/news/xliff-and-dark-mode/

Any bugs in it should be filed as (separate) bugs.
